pkg/vex/repo/repo_test.go

```diff
@@ -0,0 +1,368 @@
+package repo_test
+
+import (
+	"archive/zip"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/trivy/internal/testutil"
+	"github.com/aquasecurity/trivy/pkg/clock"
+	"github.com/aquasecurity/trivy/pkg/vex/repo"
+)
+
+var manifest = repo.Manifest{
+	Name:        "test-repo",
+	Description: "test repository",
+	Versions: []repo.Version{
+		{
+			SpecVersion: "0.1",
+			Locations: []repo.Location{
+				{
+					URL: "https://localhost",
+				},
+			},
+			UpdateInterval: repo.Duration{Duration: time.Hour * 24},
+		},
+	},
+}
+
+func TestRepository_Manifest(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Println(r.URL.Path)
```

Comment:
is this debug line?

Reply:
Absolutely. Deleted in https://github.com/aquasecurity/trivy/pull/7206/commits/07ba807deee3ea5c323ec02b51536406d13e9e84